### PR TITLE
add scite paragraph

### PR DESCRIPTION
**Pull request by vincerubinetti**

### CODE REVIEW

File: content/60.methods.md [hunk 1]
```diff
@@ -106,8 +106,11 @@ Due to the needs of this project, project contributors also implemented new feat
 Manubot uses Zotero [@url:https://www.zotero.org] to extract metadata for some types of citations.
 These features support directly citing clinical trial identifiers such as `clinicaltrials:NCT04292899` [@clinicaltrials:NCT04292899].
 
-Additionally, the fast-moving nature of the infodemic has led to a number of retractions and corrections of COVID-19 literature.
-<!--To Do: Describe integration with scite -- talk to Vince-->  
+A new plugin was also added to Manubot to support "smart citations".
+The plugin uses the [Scite](https://scite.ai/) service to display a badge below each citation with a set of icons and numbers indicating how many times that source has been mentioned, supported, or disputed.
```
Comment (rando2) [inline content/60.methods.md:110]:
```suggestion
The plugin uses the [Scite](https://scite.ai/) service to display a badge below each citation with a set of icons and numbers indicating how many times that source has been mentioned, supported, or disputed.
The scite badges were enabled in the HTML build of the document.
```

File: content/60.methods.md [hunk 2]
```diff
@@ -106,8 +106,11 @@ Due to the needs of this project, project contributors also implemented new feat
...
+The badge also indicates important editorial notices, such as retractions and corrections.
+This allows readers to roughly evaluate the reliability of cited sources at a glance.
+In this project, where we are disseminating rapidly evolving information of great consequence from over a thousand different sources, we found this feature invaluable.
```
Comment (rando2) [inline content/60.methods.md:113]:
```suggestion
In this project, where we were disseminating rapidly evolving information of great consequence from over a thousand different sources, this feature offered the invaluable function of streamlining the identification of references that needed to be double checked as time passed from when they were added.
```
I'm not quite sure if this does it, but trying to add in a mention of the temporal aspect of this challenge!

**Review by rando2 — APPROVED** [review]

File: content/60.methods.md [hunk 3]
```diff
@@ -106,8 +106,12 @@ Due to the needs of this project, project contributors also implemented new feat
 Manubot uses Zotero [@url:https://www.zotero.org] to extract metadata for some types of citations.
 These features support directly citing clinical trial identifiers such as `clinicaltrials:NCT04292899` [@clinicaltrials:NCT04292899].
 
-Additionally, the fast-moving nature of the infodemic has led to a number of retractions and corrections of COVID-19 literature.
-<!--To Do: Describe integration with scite -- talk to Vince-->  
+A new plugin was also added to Manubot to support "smart citations".
+The plugin uses the [Scite](https://scite.ai/) service to display a badge below each citation with a set of icons and numbers indicating how many times that source has been mentioned, supported, or disputed.
+The scite badges were enabled in the HTML build of the document.
```
Comment (agitter) [inline content/60.methods.md:111]:
```suggestion
The scite badges were enabled in the HTML build of the document and were available for references with a DOI.
```

It's only for DOI-based citations, right?

File: content/60.methods.md [hunk 4]
```diff
@@ -106,8 +106,12 @@ Due to the needs of this project, project contributors also implemented new feat
...
+The badge also indicates important editorial notices, such as retractions and corrections.
+This allows readers to roughly evaluate the reliability of cited sources at a glance.
```
Comment (agitter) [inline content/60.methods.md:113]:
```suggestion
This alerts readers to major reliability issues of cited sources at a glance.
```

I wouldn't want to give the impression that retractions are the main way to evaluate reliability, especially in COVID literature.

**Review by agitter — COMMENTED** [review]
Thanks for the addition.  I have couple phrasing suggestions, and the rest looks good.

File: content/60.methods.md (hunk 4 above)
Comment (vincerubinetti) [reply content/60.methods.md:113]: The "this" in that sentence was supposed to refer to the badge as a whole, not the retractions, but I didn't realize that Halie's edits made it refer to the retractions. Pushing a re-write soon.

**Review by agitter — APPROVED** [review]

**Pull request merged**